react/package.json

```diff
@@ -49,18 +52,19 @@
     "@storybook/addons": "^3.3.10",
     "@storybook/react": "^3.3.10",
     "babel-core": "^6.26.0",
+    "backstopjs": "^3.5.2",
```

Comment:
You don't actually want to install backstop in your dev dependencies.  Much better to let it come with the docker container.  It'll install PhantomJS + SlimerJS + Chrome, which is bananas.  Running it in the docker container also has the benefit of making sure we're using a consistent version of chrome with the same fonts, etc installed.